### PR TITLE
Display favorites

### PR DESCRIPTION
**Pull request by JakeGloschat**
I dont know if this will create merge conflicts...or many.  I had only made a glossaryviewmodel with code and we never touched that file.  I didn't do any work anywhere else on branch Webscrapping, just the glossaryviewmodel.

### CODE REVIEW

File: NewBrewer/NewBrewer/Views/FavoriteRecipes/FavoriteRecipesTableViewController.swift [hunk 1]
```diff
@@ -7,77 +7,78 @@
...
-class FavoriteRecipesTableViewController: UITableViewController {
 
+class FavoriteRecipesTableViewController: UITableViewController {
+    
+    // MARK: - Properties
+    var viewModel: FavoriteRecipesViewModel!
+    var beer: BeerToSave?
+    var beerFetch: Beer?
+    
+//    override func viewWillAppear(_ animated: Bool) {
```
Comment (maxpoff) [inline NewBrewer/NewBrewer/Views/FavoriteRecipes/FavoriteRecipesTableViewController.swift]: remove unnecessary code

File: NewBrewer/NewBrewer/Views/FavoriteRecipes/FavoriteRecipesTableViewController.swift [hunk 2]
```diff
@@ -7,77 +7,78 @@
...
     }
-    */
-
-    /*
-    // Override to support rearranging the table view.
-    override func tableView(_ tableView: UITableView, moveRowAt fromIndexPath: IndexPath, to: IndexPath) {
+}
 
+extension FavoriteRecipesTableViewController: FavoriteRecipesViewModelDelegate {
+//    func beerLoadedSuccessfully() {
```
Comment (maxpoff) [inline NewBrewer/NewBrewer/Views/FavoriteRecipes/FavoriteRecipesTableViewController.swift]: remoce unnecessary code

File: NewBrewer/NewBrewer/Views/FavoriteRecipes/FavoriteRecipesViewModel.swift [hunk 3]
```diff
@@ -9,20 +9,64 @@ import Foundation
...
+                
+                self.favoritedBeers.remove(at: indexOfBeer)
+                self.delegate?.beerRemovedSuccessfully()
+            case .failure(_):
+                print("Beer wasn't removed successfully.")
+            }
+        }
+    }
+    
+//    func fetchBeer(with beer: Beer) {
```
Comment (maxpoff) [inline NewBrewer/NewBrewer/Views/FavoriteRecipes/FavoriteRecipesViewModel.swift:60]: remove unnecessary code

File: NewBrewer/NewBrewer/Views/BeerIngredients/BeerIngredientsView.swift [hunk 4]
```diff
@@ -12,14 +12,19 @@ class BeerIngredientsViewController: UIViewController {
...
-        setUpTableView()
+        viewModel = BeerIngredientsViewModel(delegate: self)
     }
     
     // MARK: - Properties
     private let tableView = UITableView()
     private let tableViewCellReuseIdentifier = "TableViewCell"
     private let collectionViewCellReuseIdentifier = "CollectionViewCell"
-    var beer: Beer?
+    var beer: Beer? {
```
Comment (maxpoff) [inline NewBrewer/NewBrewer/Views/BeerIngredients/BeerIngredientsView.swift:22]: what a cool didSet

**Review by maxpoff — APPROVED** [review]
Fix comments and you're good to go

File: NewBrewer/NewBrewer/Views/BeerIngredients/BeerIngredientsView.swift (hunk 4 above)
Comment (JakeGloschat) [reply NewBrewer/NewBrewer/Views/BeerIngredients/BeerIngredientsView.swift:22]: I know, right?  Learned from the best.

**Pull request merged**